### PR TITLE
Closes #https://github.com/kestra-io/kestra/issues/4586 New page to show user a message that the flow doesn't have dependencies

### LINKED ISSUE #4586: No Dependencies, display a proper page
When the flow don't have any dependencies, we are displaying this: 
image

Replace this by removing the graph and display a proper page, something similar to: 
image
- no button 
- a message: "Your flow don't have any dependencies"
- an image not afraying

### PR DESCRIPTION
**Pull request by mounikaperi**
### What changes are being made and why?

New page to show user a message that the flow doesn't have dependencies
Resolves: https://github.com/kestra-io/kestra/issues/4586
Currently, when there are no dependencies for a workflow we are showing an empty graph
What have we changed?
Now, if the dependencies length is greater than 0, we will go ahead and show the existing graph component, else we will be showing a page which says "Your flow don't have dependencies" and a documentation link which gives more information

Test Results:
Dark Theme
DocumentationsDarkTheme
Light Theme
DocumentationsLightTheme

---

### How the changes have been QAed?

---

### CODE REVIEW

**Review by MilosPaunovic — CHANGES_REQUESTED** [review]
@mounikaperi Could you please remove all the changes to every translation file except `en.json`, as we have a Ci workflow which automatically adds a PR once the code is merged to `develop` to add missing key/value pairs to languages other than English, based on some of our rules.

And also, the second line of text should not be in primary color, but white/dark (based on mode) with a link only in primary colors.

**Comment by mounikaperi** [comment]
@MilosPaunovic Sure, i will incorporate above review comments and will let you know

**Comment by mounikaperi** [comment]
@MilosPaunovic pushed the changes. Can you please check now

File: ui/src/components/flows/FlowNoDependencies.vue [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    <div class="no-dependencies-container">
+        <div>
+            <img :src="flowImage" alt="No dependencies">
+        </div>
+        <div class="no-dependencies-message">
+            <p>{{ $t("flow no dependencies") }}</p>
+        </div>
+        <div class="no-dependencies-doc-message" :class="themeClass">
+            <p>
+                {{ $t("Read more about") }}
```
Comment (MilosPaunovic) [inline ui/src/components/flows/FlowNoDependencies.vue]: You need to add this key to the `en.json` file.

File: ui/src/components/flows/FlowNoDependencies.vue [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                <a
+                    :href="dependenciesDocsUrl"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    class="no-dependencies-doc-message doc-link"
+                >
+                    {{ $t("flow-dependencies") }}
+                </a>
+                {{ $t("in our documentation.") }}
```
Comment (MilosPaunovic) [inline ui/src/components/flows/FlowNoDependencies.vue]: This key as well.

**Review by MilosPaunovic — CHANGES_REQUESTED** [review]

**Comment by mounikaperi** [comment]
@MilosPaunovic done

**Review by MilosPaunovic — APPROVED** [review]
LGTM! 🚀

**Pull request merged**